package.json

```diff
@@ -44,16 +44,17 @@
   },
   "dependencies": {
     "7zip-bin": "^5.0.3",
+    "aws-sdk": "^2.1044.0",
     "chokidar": "^3.5.0",
     "electron-is-dev": "^1.2.0",
     "electron-log": "^4.3.1",
     "electron-settings": "^3.2.0",
     "electron-updater": "^4.3.5",
     "extract-zip": "^2.0.1",
+    "gist-client": "^1.1.1",
     "got": "^11.8.2",
     "node-7z": "^2.1.2",
-    "octonode": "^0.10.2",
-    "sdfz-demo-parser": "^4.1.2",
+    "sdfz-demo-parser": "^4.6.0",
```

Comment:
can't just add random changes like this.

Reply:
I think it was introduced by 'npm audit fix' to resolve some serious vulnerability it was complaining about.

Reply:
I think it was introduced by including various other changes from bar/spring-launcher, not necessarily something you did